File: ass1_q1.py
#1811090
print('Programme to print sum of 1 to n integers')

def sum(n):
    s = 0
    for i in range(n+1):
        s += i
    if s == 0:
        print('Please enter the positive number')
    else:
        print(f'Sum of 1 to {n} integers is: {s}')


n = int(input('Insert the value of integer n: '))
sum(n)

sum(100)

File: ass1_q1.cpp
#include <iostream>
using namespace std;

int main()
{   
    cout << "Programme to print sum of 1 to n integers" <<endl;
    cout << "Enter a integer n: ";

    int n, sum = 0;
    cin >> n;

    if (n <= 0){
        cout << "Please enter the positive number" <<endl;
    }
    else{
        for (int i = 1; i <= n; ++i){
            sum += i;
        }
        cout << "Sum of 1 to "<< n << " is " << sum <<endl;
    }

    return 0;
}